clusterctl/examples/vsphere/machines.yaml.template

```diff
@@ -0,0 +1,62 @@
+items:
```

Comment:
Now that you are using the vsphere client, this may need to updated based on PR #257. The apiVersion will be vsphereproviderconfig and the kind is either VsphereClusterProviderConfig or VsphereMachineProviderConfig. The username/password/server fields moved from the machine to the cluster providerConfig.

Reply:
Done.